es-core/src/components/ImageGridComponent.h

```diff
@@ -90,27 +140,246 @@ class ImageGridComponent : public IList<ImageGridData, T>
 
 	bool mEntriesDirty;
 
-	std::vector<ImageComponent> mImages;
+	Eigen::Vector2f mMargin;
+	Eigen::Vector2i mDesiredGridSize;
+
+	const int MAX_TEXTURES = 80;			// The maximum amount of images that can be loaded at once
```

Comment:
Where does the magic number come from?

Reply:
From my magical hat ... I have no idea for this one

Reply:
This look like a security check to prevent leaks when the dynamic image loader load too many images in very specific case.